### PR TITLE
add integrations

### PR DESCRIPTION
**Pull request by pi0**
Preview:

- https://deploy-preview-605--nuxt.netlify.app/resources/integrations

DB: https://github.com/nuxt/integrations

Co-authored-by: Debbie O'Brien <debbie@nuxtjs.com>

### CODE REVIEW

File: components/TheAsideNavResources.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <div
+      class="h-full overflow-y-auto scrolling-touch text-center lg:text-left lg:h-auto lg:block lg:relative lg:sticky lg:top-24"
+    >
+      <nav
+        class="pt-8 lg:overflow-y-auto lg:block lg:pl-0 lg:pr-8 sticky?lg:h-(screen-24)"
+      >
+        <h2
+          class="text-xl xl:text-2xl text-light-onSurfacePrimary dark:text-dark-onSurfacePrimary font-medium leading-normal mb-4 lg:pt-4"
+        >
+          Resourcs
```
Comment (danielroe) [inline components/TheAsideNavResources.vue:14]:
```suggestion
          Resources
```

File: content/en/guides/internals-glossary/context.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ Use this method to redirect the user to another route, the status code is used o
 
 Use this method to show the error page: `error(params)`. The `params` should have the properties `statusCode` and `message`.
 
-### $config
+### \$config
```
Comment (danielroe) [inline content/en/guides/internals-glossary/context.md:104]:
```suggestion
### $config
```

File: components/TheAsideNavResources.vue (hunk 1 above)
Comment (pi0) [reply components/TheAsideNavResources.vue]: Too early review :P

Comment (danielroe) [reply components/TheAsideNavResources.vue]: Sorry! Didn't mean it to be a review, just a passing comment....! 😬

File: pages/integrations/_name.vue [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+          </template>
+          <template v-if="integration.npm">
+            <a
+              v-for="type in ['v', 'dm', 'types']"
+              :key="type"
+              :href="`https://npmjs.com/package/${integration.npm.name}`"
+              target="_blank"
+            >
+              <img
+                :src="`https://flat.badgen.net/npm/${type}/${integration.npm.name}`"
```
Comment (manniL) [inline pages/integrations/_name.vue:81]: It seem like `integration.npm.name` is undefined sometimes (e.g. on https://deploy-preview-605--nuxt.netlify.app/integrations/netlify-files )

Comment (pi0) [reply pages/integrations/_name.vue:81]: Actually it was refactored to simple string for `npm` field. I forgot to update badges

Comment (manniL) [reply pages/integrations/_name.vue:81]: Will change it accordingly then :)

Comment (manniL) [reply pages/integrations/_name.vue:81]:
```suggestion
                :src="`https://flat.badgen.net/npm/${type}/${integration.npm}`"
```

File: pages/integrations/_name.vue [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                :alt="'github ' + type"
+                class="mb-2"
+              />
+            </a>
+          </template>
+          <template v-if="integration.npm">
+            <a
+              v-for="type in ['v', 'dm', 'types']"
+              :key="type"
+              :href="`https://npmjs.com/package/${integration.npm.name}`"
```
Comment (manniL) [inline pages/integrations/_name.vue:77]:
```suggestion
              :href="`https://npmjs.com/package/${integration.npm}`"
```

**Comment by manniL** [comment]
As a side note - we might want to add meta tags (e.g. title & description) for the integrations. These are currently missing.

**Comment by pi0** [comment]
@manniL Metadata is kept in [nuxt/integrations](https://github.com/nuxt/integrations) and automatically syncable with upstream repositories :)

File: preview-image.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+const puppeteer = require('puppeteer')
+
+;(async () => {
```
Comment (febrihidayan) [inline preview-image.js:3]: I just found out there is such a way 😁

Comment (debs-obrien) [reply preview-image.js:3]: Haha spying on our work in progress. Yes there is always a way 😄

File: pages/resources/integrations/_category.vue [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+<script>
+import uniq from 'lodash/uniq'
+
+export default {
+  transition: 'resources',
+  async asyncData({ $content, params }) {
+    const rawCategories = await $content('integrations')
+      .only(['name', 'categories'])
+      .fetch()
+    const categories = uniq(
```
Comment (manniL) [inline pages/resources/integrations/_category.vue:46]: Could we possibly use `new Set`  instead of `uniq` here (should save a few bytes on the bundle)?

**Pull request closed without merging**